packages/shared-lib/api/user.ts

```
@@ -59,3 +60,120 @@ export const updateUserData = async (param: UpdateUserParams) => {
+  switch (category) {
+    case "0":
+      const userList = (await client.models.User.list()).data;
+      switch (order) {
+        case "latest":
+          return userList.sort((a, b) => {
+            const dateA = a.createdAt ? new Date(a.createdAt).getTime() : 0;
+            const dateB = b.createdAt ? new Date(b.createdAt).getTime() : 0;
+            return dateB - dateA;
+          });
+        case "oldest":
+          return userList.sort((a, b) => {
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)